src/app.css

```
@@ -54,5 +85,7 @@
 	}
 	body {
 		@apply bg-background text-foreground;
+		font-family: 'Open Sans', sans-serif;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)